client_example/22_grouped_gemm/grouped_gemm_fixed_nk_fp8.cpp

```diff
@@ -63,8 +63,8 @@ int main()
     for(int i = 0; i < group_count; ++i)
     {
         Ms.push_back(256 + 256 * i);
-        Ns.push_back(128 + 128 * i);
-        Ks.push_back(128 + 64 * i);
+        Ns.push_back(128);
+        Ks.push_back(128);
```

Comment:
You have to update Kbatch value also in this example.

Reply:
Why?